### DISCUSSION TITLE
Include Procaptcha in Cryptographic challenges

### DISCUSSION THREAD

**Discussion by HughParry** [relaycorp/ddos-report#21] [General]
Would it be possible to include Procaptcha (https://prosopo.io/ | https://github.com/prosopo/captcha) alongside Altcha and mCaptcha in the Turnkey PoW solutions section here?

https://ddos.report/tactics/crypto-challenges/

**Comment by gnarea** [comment]
Hi @HughParry!

Has Prosopo reached General Availability? I'm asking because your roadmap has entries for the "alpha" and "beta" phases, but nothing about GA or "stable".

If that's the case, I'd be happy to include it. Otherwise, I'd rather wait until it reaches GA.

**Comment by HughParry** [comment]
Hey @gnarea, thanks for getting back to me!

Yeah we've reached general availability - if you want to integrate with your site then I'd be happy to lend a hand with it! 

Thanks for the feedback on our roadmap btw - I've removed that from the website since it's a bit out of date and the messaging is confusing.

**Reply by gnarea** [reply]
👍🏾 

Done! https://github.com/relaycorp/ddos-report/commit/c6920af0cba6a9edbe52a90267612450c22f1997
